PokeAlarm/Filters/MonFilter.py

```diff
@@ -215,6 +221,10 @@ def to_dict(self):
         if self.max_dist is not None:
             settings['max_dist'] = self.max_dist
 
+        # Pokemon types
+        if self.type_ids is not None:
+            settings['type_ids'] = self.type_ids
```

Comment:
Please include a test for this filter in https://github.com/pokealarm/pokealarm/blob/dev/tests/filters/test_monster_filter.py to ensure it properly works and evaluates before merger.